.config-schema.json

```diff
@@ -13,15 +13,15 @@
             "x-env-variable": "OPENFGA_MAX_TYPES_PER_AUTHORIZATION_MODEL"
         },
         "maxConcurrentReadsForCheck": {
-            "description": "The maximum allowed number of concurrent reads in a single Check query. The default is MaxUint32",
+            "description": "The maximum allowed number of concurrent reads in a single Check query (default is MaxUint32).",
             "type": "integer",
             "default": 4294967295,
             "x-env-variable": "OPENFGA_MAX_CONCURRENT_READS_FOR_CHECK"
         },
         "maxConcurrentReadsForListObjects": {
-            "description": "The maximum allowed number of concurrent reads in a single ListObjects query.",
+            "description": "The maximum allowed number of concurrent reads in a single ListObjects query (default is MaxUint32).",
             "type": "integer",
-            "default": 30,
+            "default": 4294967295,
```

Comment:
This is a breaking behaviour - do we have consensus of tweaking this value

Reply:
So was the unintended breaking change to the concurrency limits in Read.

It's new behavior. We got the values wrong.

Reply:
In this case, please
- ensure it is well documented in the changelog 
- update title of PR to reflect that we are changing the default behaviour of listobject

Reply:
Do we also want to update the sample docker-compose file and https://openfga.dev/docs/getting-started/running-in-production to recommend operator setting a reasonable value

Reply:
I will update the production best practices documentation, but I think leaving the docker-compose file as is would be fine. If you and @miparnisari feel strongly otherwise, I'm happy to update the docker-compose file as well.

Consensus?